contracts/crowdsale/distribution/RefundableCrowdsale.sol

```diff
@@ -21,25 +21,37 @@ contract RefundableCrowdsale is FinalizableCrowdsale {
   // refund vault used to hold funds while crowdsale is running
   RefundVault public vault;
 
-  function RefundableCrowdsale(uint256 _goal) public {
+  /**
+   * @param _goal Funding goal
+   * @param _vault Refund vault to temporarily store funds
+   */
+  function RefundableCrowdsale(uint256 _goal, RefundVault _vault) public {
     require(_goal > 0);
-    vault = new RefundVault(wallet);
+    require(_vault != address(0));
```

Comment:
What is the motivation of this change? If it's not the crowdsale who deploys the RefundVault, it's not guaranteed that the crowdsale is going to be its owner. This requires trust in the deployer that it will set the crowdsale as the  vault owner, as it can steal all the funds anytime by providing a modified `RefundVault` version where anyone (not only the owner) can call `RefundVault.deposit`, allowing this crowdsale to send its funds to it to be stolen.

Reply:
Ah, it was to lower the deployment cost?

Reply:
If it was to lower deployment costs, maybe the `RefundVault` can be added in a post-construction stage, but I don't want for it to get too complex.